tests/test_climate_baseline_helpers.py

```diff
@@ -45,10 +43,19 @@ def histories(stations):
     ) for station in stations for year in [2000, 2001]]
 
 
+@fixture
+def sesh_with_station_and_history_records(sesh_with_climate_baseline_variables, stations, histories):
+    for sesh in generic_sesh(sesh_with_climate_baseline_variables, stations + histories):
+        yield sesh
+
+
+climatology_var_names = ['Tx_Climatology', 'Tn_Climatology', 'Precip_Climatology']
+
+
 def describe_get__or__create__pcic__climate__variables__network():
 
-    def test_creates_the_expected_new_network_record(empty_database_session):
-        sesh = empty_database_session
+    def test_creates_the_expected_new_network_record(session):
+        sesh = session
```

Comment:
Eh, you barely use the local variable sesh... 1-2x per test. Is it really worth the aliasing? Likewise for the next several tests.